### PR TITLE
UM-036 - Energy Gun/Cell Adjustments 

### PR DESCRIPTION
**Pull request by UrsulaMejor**
## About the PR 

Taser bolts now cost 25PU
Lasers now cost 31.25PU
Wave gun projectile costs increased to 50 for default, 75 for transverse, and 150 for EMP
New Taser Shotgun projectile made to preserve old behavior

Tasers now start with Medium Power cells (200PU) installed, which is standard to all other sec equipment
Eguns now start with Medium+ Power cells (250PU) installed, a new power cell type
Laser Guns now start with Medium+ Power cells (250PU) installed
Wave Guns now start with High Power (300) PU power cells installed

Security pouches now start with a High Power cell (300PU) instead of a Medium Power cell. This allows them to upgrade either their taser OR their baton with the cell.

Net results:
Default tasers can shoot 8 bolts instead of 6. Upgraded tasers shoot 12 bolts instead of 13.
Default stun batons have 8 uses, same as before. Upgraded stun batons have 12 uses.
Default eguns can shoot 10 taser bolts instead of 13
Default eguns can shoot 8 laser bolts, same as before
Default laser guns can shoot 8 laser bolts, same as before
Default taser shotguns can shoot 20 taser bolts, same as before
Default taser shotguns can shoot 8 spread shots, same as before.
Default waveguns can shoot 6 shots default, 4 shots transverse, and 2 shots EMP, same as before.

## Why's this needed? 

Evens out some of the wonky power cell costs and values and gives the opportunity for security to upgrade more than just their taser if they have a different preference.

## Changelog

```
(u)UrsulaMajor:
(+)Security starts with a larger power cell in their pouch; taser, egun, and laser gun cell sizes and projectile costs adjusted to compensate
```

### CODE REVIEW

File: code/obj/item/gun/ammo.dm [hunk 1]
```diff
@@ -937,6 +937,16 @@
 	charge = 200.0
 	max_charge = 200.0
 
+/obj/item/ammo/power_cell/med_plus_power
+	name = "Power Cell - 200"
```
Comment (Tarmunora) [inline code/obj/item/gun/ammo.dm:941]:
```suggestion
	name = "Power Cell - 250"
```

**Comment by Tarmunora** [comment]
Note that this does change taser shotguns, as they have a base taser shot option - 8 shots of spread or 12 shots of taser after this change
Additionally, I would recommend increasing wavegun cell to 300 and adjusting costs somewhat if giving sec pouches a 300u cell roundstart

**Comment by UrsulaMejor** [comment]
> 
> 
> Note that this does change taser shotguns, as they have a base taser shot option - 8 shots of spread or 12 shots of taser after this change
> Additionally, I would recommend increasing wavegun cell to 300 and adjusting costs somewhat if giving sec pouches a 300u cell roundstart

oops, I misread the taser shot subtype for azungar's thingy and thought that the child overrode the cost of the parent. I can fix that real quick.

Wavegun stuff noted, will follow up in a bit

**Comment by Tarmunora** [comment]
Used to get 20 shots of taser 🤷

**Comment by UrsulaMejor** [comment]
what to heck, that's a lot D:

**Pull request merged**